### PR TITLE
Update Azure table offerings for application Architecture

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

- Update recommendations
- Convert it to markdown for accessibility

### CODE REVIEW

**Comment by mvelosop** [comment]
@nishanil please check the _by-default_ recommendations.

**Comment by mairaw** [comment]
The images are slightly different in size. 

image

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md [hunk 1]
```diff
@@ -19,9 +19,19 @@ However, this recommendation should be taken with a pinch of salt, as the produc
...
+| **Batch/Jobs (Background tasks)** | ![Possible with VMs](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Possible with ACI](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Possible with App Service](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Possible with AKS](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Recommended with Azure Functions](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> Background tasks | ![Recommended with Azure Batch](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> Large scale |
+
+**Legend**
+
+![Recommended](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) Recommended \
```
Comment (mairaw) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]:
```suggestion
![Recommended icon](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) Recommended \
```

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md [hunk 2]
```diff
@@ -19,9 +19,19 @@ However, this recommendation should be taken with a pinch of salt, as the produc
...
+![Possible](media/choosing-azure-compute-options-for-container-based-applications/possible.png) Possible
```
Comment (mairaw) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]:
```suggestion
![Possible icon](media/choosing-azure-compute-options-for-container-based-applications/possible.png) Possible
```

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md [hunk 3]
```diff
@@ -19,9 +19,19 @@ However, this recommendation should be taken with a pinch of salt, as the produc
 
 After a deeper analysis of the application’s needs, the product selected could be different. But, as a starting point, it is good to have initial guidance from where you can start evaluating and testing based on certain priority.
 
-In Figure 1, you can see a breakdown of different kinds of apps and their ideal Azure hosting scenarios.
+In the next table, you can see a breakdown of different kinds of apps and their possible and recommended Azure hosting scenarios.
```
Comment (mairaw) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]:
```suggestion
In the following table, you can see a breakdown of different kinds of apps and their possible and recommended Azure hosting scenarios.
```

**Comment by mvelosop** [comment]
> The images are slightly different in size.
> 
> image

Yes, I wanted to make them highlight that way, besides the color, that they're the recommended option.

Did you like the white outline to improve the contrast in dark mode?

Just experimenting 😊

**Comment by nishanil** [comment]
@mvelosop Can you make the images smaller.. And, update ms.date.

File: docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md [hunk 4]
```diff
@@ -19,9 +19,19 @@ However, this recommendation should be taken with a pinch of salt, as the produc
...
+| **N-Tier apps (Services)**        | ![Possible with VMs](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Possible with ACI](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Recommended with App Service](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) | ![Possible with AKS](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | ![Possible with Azure Fuctions](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | |
+| **Cloud-Native (Microservices)**  | | ![Possible with ACI](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | | ![Recommended with AKS](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> for Linux | ![Recommended with Azure Functions](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> Event-Driven | |
```
Comment (nishanil) [inline docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]:
```suggestion
| **Cloud-Native (Microservices)**  | | ![Possible with ACI](media/choosing-azure-compute-options-for-container-based-applications/possible.png) | | ![Recommended with AKS](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> (Linux containers) | ![Recommended with Azure Functions](media/choosing-azure-compute-options-for-container-based-applications/recommended.png) <br/> Event-Driven | |
```

Comment (nishanil) [reply docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]: I think for comments within the table, it's better to use parenthesis `( .. )`

Comment (mvelosop) [reply docs/architecture/modernize-with-azure-containers/modernize-existing-apps-to-cloud-optimized/choosing-azure-compute-options-for-container-based-applications.md]: Done.

**Review by nishanil — APPROVED** [review]
LGTM

**Pull request merged**